metaflow/plugins/argo/argo_workflows.py

```diff
@@ -469,6 +474,11 @@ def _compile_workflow_template(self):
                 }
             )
 
+        # get labels from env vars
```

Comment:
If you want this to be a method of some sort let me know.  This seems to add the labels to pods.  If you think it should be somewhere else, let me know.  I am not that familiar with the argo plugin since I was previously relying on the fact that the kubernetes decorator would automatically add labels for us.